src/components/StakeQuickModal/StakeQuickModal.tsx

```diff
@@ -146,19 +136,17 @@ const StakeQuickModal: React.FC<StakeQuickModalProps> = ({ open, onClose }) => {
             </Typography>
           </Box>
           <Box mt={2} display='flex' alignItems='center'>
-            <input
+            <NumericalInput
               placeholder='0'
-              className={classes.input}
               value={typedValue}
-              onChange={(evt: any) => {
+              fontSize={28}
+              onUserInput={(value) => {
```

Comment:
Now that were using a numerical input can't value be typed already?

Reply:
The value is returning the typed string in the input